### PR TITLE
Brian branch

### PR DESCRIPTION
**Pull request by coatexblee**
Before  you try to merge/rebase my branch to your branch, please make sure your master branch is already updated by nate_branch( before been merge), and both of your branches should have the same contents.  Also double check if there  any suspicious my codes could overwrite your codes. If nate_branch runs without any problem, update your master branch from nate_branch. I'll fetch your master branch and rebase it into my repository, thus updating my old master, then work more on updating the code. Then the recycle continue.

### CODE REVIEW

File: demographics.html [hunk 1]
```diff
@@ -67,150 +65,102 @@ <h1>Demographics</h1>
...
-                    </div>
-                </div>
-            </div>
             <div class="row">
                 <div class="col-md-6 col-sm-6 col-xs-12">
                     <h4 align="center"> Housing Tenure </h4>
-                    <div id="canvas-holder2" align="center">
-                        <canvas id="housingCanvas" width="400" height="300" />
+                    <div id="canvas-holder" align="center">
+                        <canvas id="canvas3" width="400" height="300" />
```
Comment (NateATXCode) [inline demographics.html]: this looks exactly the same as the previous pull request.

File: demographics.html [hunk 2]
```diff
@@ -67,150 +65,102 @@ <h1>Demographics</h1>
...
-    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
-    <script src="https://d3js.org/d3.v3.min.js" charset="utf-8"></script>
-    <!-- js files -->
-    <script src="js/chart.js"></script>
-    <script src="js/AutoGenerateCharts_v2.js"></script>
-    <script type="text/javascript" src="js/custom-script.js"></script>
+    </div>  <!-- /container -->
+
+    <script src="js/AutoGenerateCharts.js"></script>
```
Comment (NateATXCode) [inline demographics.html]: this is the old JS file.

this removed all the old JS.

**Comment by coatexblee** [comment]
Thank you for checking. Just to be sure.

**Review by NateATXCode — COMMENTED** [review]
you have an old demographics.html page here. this is similar to the last PR

**Comment by coatexblee** [comment]
Yes, it is similar because I’m testing with other files at a time to see if my developer branch could merge with your developer branch.  That will come later. Did you  check nate_branch with your master_branch to see if it is equal? Ignore my master branch.  The actual changes is in my developer branch aka brian_branch.  Did you notice the differences when you open HTML files in any two of your branches vs brian_branch?

**Comment by NateATXCode** [comment]
master = nate_branch

**Comment by coatexblee** [comment]
I confirmed in the main repository. brian_branch should have few modified HTML files. My master branch should be same as your two branches.

**Pull request closed without merging**